plop-templates/github-release.yml.hbs

```diff
@@ -28,11 +28,6 @@ jobs:
         run: npx semantic-release
       {{/internal}}
       {{#internal}}
-      - name: Setup Node.js
-        uses: actions/setup-node@v2
-        with:
-          node-version: "16.x"
-          registry-url: "https://npm.pkg.github.com"
```

Comment:
The internal registry still has to be set up for internal modules or the installation will fail if someone tried to install a @seamapi/* package